src/components/application_manager/src/commands/mobile/register_app_interface_request.cc

```diff
@@ -638,6 +638,8 @@ void RegisterAppInterfaceRequest::SendRegisterAppInterfaceResponseToMobile() {
   SendOnAppRegisteredNotificationToHMI(
       *(application.get()), resumption, need_restore_vr);
 
+  // Start PTU after successfull registration
+  GetPolicyHandler().AddApplication(application->policy_app_id());
```

Comment:
@dev-gh I'm not sure that it will be more suitable - after starting PTU we should send OnApplicationRegistered and then OnPermissionChange, so if I move AddApplication to OnAppRegisteredOnMobile then I have to move also OnApplicationRegistered, or the flow will become invalid.